xdem/coreg/affine.py

```diff
@@ -2391,6 +2395,8 @@ def __init__(
         :param bin_statistic: Statistic of central tendency (e.g., mean) to apply during the binning.
         :param subsample: Subsample the input for speed-up. <1 is parsed as a fraction. >1 is a pixel count.
         :param vertical_shift: Whether to apply the vertical shift or not (default is True).
+        :param initial_shift: Tuple containing x, y and z shifts (in georeferenced units).
+        These shifts are applied before the fit() part.
```

Comment:
Need to add a tabulation here, or the line won't appear as part of the previous `:param` :wink: